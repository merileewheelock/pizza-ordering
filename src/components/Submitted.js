import React, { Component } from 'react';

class Submitted extends Component{
	render(){
		return(
			<div>
				<h1 className="submitted-header text-center">Your order is on its way!</h1>
				<div className="text-center"><img src="/images/delivery-man.png" alt="deliver-man" /></div>
			</div>
		)
	}
}

export default Submitted;