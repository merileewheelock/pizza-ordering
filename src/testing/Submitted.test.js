import React from 'react';
import Submitted from '../components/Submitted';
import TestRenderer from 'react-test-renderer'; // ES6

it('renders correctly', () => {
    const tree = TestRenderer
        .create(<Submitted></Submitted>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});