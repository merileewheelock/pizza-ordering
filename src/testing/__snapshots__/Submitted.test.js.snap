// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1
    className="submitted-header text-center"
  >
    Your order is on its way!
  </h1>
  <div
    className="text-center"
  >
    <img
      alt="deliver-man"
      src="/images/delivery-man.png"
    />
  </div>
</div>
`;
